check_rpmspec_collection.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+if [ "$1" = "-h" -o "$1" = "--help" ]; then
+    echo "Description - Verify rpm package count with various combination of ansible and collection_artifact options."
+    echo "            - Check there is no README.html files in /usr/share/andible/collections."
+    echo "Usage: $0 [ -h | --help ] | [ fedpkg | rhpkg [ branch_name ] ]"
+    exit 0
+fi
+
+if [ ! -f *.spec ]; then
+    echo "You have to have an rpm spec file in the current directory."
+    exit 1
+fi
+
+set -euo pipefail
+
+pkgcmd=${1:-"fedpkg"}
+branch=${2:-"rawhide"}
+run() {
+    local option=$1
+    local ext=$2
+    local expect=$3
+    \rm -rf noarch
+    output=/tmp/$pkgcmd.$ext
+    cmdline="$pkgcmd --release $branch local $option"
+    echo $cmdline
+    $cmdline > $output 2>1
+    count=$( ls noarch | wc -l )
+    if [ $count -eq $expect ]; then
+      echo OK - result $count is $expect
+    else
+      echo FAILED - result $count is not $expect
+    fi
+
+    filename=$( $pkgcmd --release $branch verrel )
+    rval=$( rpm -ql noarch/${filename}*.rpm | egrep '.html' | egrep collection | egrep -v '/usr/share/doc/' || : )
```

Comment:
you don't need `rpm -qlp` here?  I guess rpm is now smart enough to know that the given argument is the name of a package filename instead of a package.

Reply:
did you mean `fgrep` (literal match e.g. match the literal `.` in `.html` instead of treating `.` as "match any char") here instead of `egrep` (extended regular expression match)?